### PR TITLE
Rename plot function

### PR DESCRIPTION
**Pull request by bailliem**
* Rename plot to visR
* Replace all instances of visR::plot with visR::visr
* rename plot.R to plot_visr.R
* tidy up documentation

### CODE REVIEW

**Review by SHAESEN2 — CHANGES_REQUESTED** [review]
If we are going to be consistent I would also change:
- utils_plot.R => utils_visr.R

File: R/visr_plot.R [hunk 1]
```diff
@@ -101,13 +109,13 @@ plot.survfit <- function(
       log = "log(Survival probability)",
       event = "Failure probability",
       cloglog = "log(-log(Survival probability))",
-      pct = "Survival probability (%)",
-      logpct = "log(Survival probability (%))",
+      pct = "Survival percentage (%)",
```
Comment (SHAESEN2) [inline R/visr_plot.R:112]: Not sure if this is the correct term.

**Review by SHAESEN2 — APPROVED** [review]
Not sure if  'survival percentage' is the correct wording. Perhaps Percentage survival might be better?

File: R/visr_plot.R (hunk 1 above)
Comment (bailliem) [reply R/visr_plot.R:112]: Agree. Survival percentage makes more sense. Also we can then drop the %.

Comment (bailliem) [reply R/visr_plot.R:112]: Same for log surv percentage

Comment (SHAESEN2) [reply R/visr_plot.R:112]: Did you mean Percentage survival makes more sense? (survival percentage is what it is saying now)

Comment (bailliem) [reply R/visr_plot.R:112]: Percentage survival.

**Comment by SHAESEN2** [comment]
Still some weird phrasing present. Cloglog is not in percentage and you forgot to change one survival percentage

**Comment by bailliem** [comment]
Remove theme_bw from visR

**Pull request merged**